library/src/main/java/com/prolificinteractive/materialcalendarview/format/DateFormatDayFormatter.java

```diff
@@ -0,0 +1,34 @@
+package com.prolificinteractive.materialcalendarview.format;
+
+import com.prolificinteractive.materialcalendarview.CalendarDay;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Locale;
+
+/**
+ * Format using a {@linkplain DateFormat} instance.
+ */
+public class DateFormatDayFormatter implements DayFormatter {
+
+    private final DateFormat dateFormat;
+
+    /**
+     * Uses "MMMM yyyy" for formatting
+     */
+    public DateFormatDayFormatter() {
+        this.dateFormat = new SimpleDateFormat("d", Locale.getDefault());
```

Comment:
Single "d" will create the shortest day of the month string. I should probably fix the javadoc for this constructor though.